src/web/routes/application/confirm.test.js

```diff
@@ -0,0 +1,93 @@
+const test = require('tape')
+const sinon = require('sinon')
+const proxyquire = require('proxyquire')
+const { ELIGIBLE } = require('./common/constants')
+
+const wrapError = sinon.spy()
+
+const { toPounds, isNilOrLessThanZero, getConfirmPage } = proxyquire('./confirm', {
+  './common/formatters': { wrapError }
+})
+
+test('toPounds() converts value in pence to pounds', (t) => {
+  const expected = '3.10'
+  const result = toPounds(310)
+  t.equal(result, expected, 'gets voucher value in pence and converts to pounds')
+  t.end()
+})
+
+test('isNilOrLessThanZero()', (t) => {
```

Comment:
Can you put a test for zero in here too? I guess you missed that because of the method name not being quite right.